packages/@aws-cdk/aws-appsync/lib/resolver.ts

```diff
@@ -18,9 +19,10 @@ export interface BaseResolverProps {
   /**
    * configuration of the pipeline resolver
    *
-   * @default - create a UNIT resolver
+   * @default - no pipeline resolver configuration
+   * An empty array | undefined sets resolver to be of kind, unit
    */
-  readonly pipelineConfig?: CfnResolver.PipelineConfigProperty | IResolvable;
+  readonly pipelineConfig?: string[];
```

Comment:
can this be more strongly typed? or is there a good reason why strings would be ideal?

this motivation for this is not covered in the commit body, can we add that?
it's especially important since this is a breaking change (also needs mention in commit body)

Reply:
mmmm originally the type would have been this:
```
{
  functions: [...]
}
```

where the functions parameter just took a `string []` anyways, so changed it to just a `string []` after following Mitch's suggestion.

Reply:
also is this a breaking change?

the `pipelineConfig` prop doesn't actually do anything

Reply:
do we have an issue for pipelineConfig doing nothing?

consider this:
* user has a  CDK application where they have specified a pipelineConfig of type `PipelineConfigProperty`

after this change, when they "upgrade" and just build.. it will fail saying the type is not a match. They have not changed anything about their application other than upgrade, but their build is now failing.

I think it would break which is why I'd consider this to be a breaking change. what do you think?

Reply:
@shivlaks issue is here https://github.com/aws/aws-cdk/issues/6923

Flattening out the object is a convenience I think since it's the only property. Technically cfn could add more optional properties in the future and we would have to handle those but that happens either way.

I think the more strongly typed could be 'IFunction[]'. @BryanPan342 does that make sense to you?